Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23005,5 +23005,105 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Show sales in the week 3 of 2027",
+    "Context": {
+      "ReferenceDateTime": "2018-07-02T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "week 3 of 2027",
+        "Start": 18,
+        "End": 31,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2027-W03",
+              "type": "daterange",
+              "start": "2027-01-18",
+              "end": "2027-01-25"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Show sales in the week 27 of last year",
```

Comment:
Can you add a case for "Let's meet on Thursday, week 23"?

Reply:
added